molgenis/molgenis-emx2
# chore: add error handling on page not found

Now show error instead of infinite spinner when navigating to a biobank/collection/service/study that doesn't exist

### What are the main changes you did
- Added more error handling

### How to test
- Go to an invalid biobank e.g. https://preview-emx2-pr-4573.dev.molgenis.org/directory-demo/directory/#/biobank/invalid
- See error
- Go to an invalid collection e.g. https://preview-emx2-pr-4573.dev.molgenis.org/directory-demo/directory/#/collection/invalid
- See error
- Go to an invalid study e.g. https://preview-emx2-pr-4573.dev.molgenis.org/directory-demo/directory/#/study/invalid
- See error
- Go to an invalid service e.g. https://preview-emx2-pr-4573.dev.molgenis.org/directory-demo/directory/#/service/invalid
- See error

### Checklist
